### PR TITLE
feature/DPP-120

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

**Review by maikel-maas — COMMENTED** [review]
LGTM but someone else needs to approve it since I'm the author of the PR

File: packages/ssi-react/src/components/views/SSITableView/PaginationControls/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const PaginationControls: FC<PaginationControlsProps> = (props: PaginationControlsProps): ReactElement => {
+  const {onKeyDown, containerStyle, goToInputId = 'goTo'} = props
+  return (
+    <PaginationContainerStyled style={containerStyle}>
+      <Stack>
+        <PaginationStyled {...props} />
+      </Stack>
+      <GoToInputContainer>
+        Go to <GoToInputStyled id={goToInputId} onKeyDown={onKeyDown} /> page
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/PaginationControls/index.tsx:18]: we need to add the localization

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 2]
```diff
@@ -46,6 +47,7 @@ type Props<T> = {
   enableResultCount?: boolean
   columnResizeMode?: ColumnResizeMode
   actions?: Array<Button>
+  paginationControlsProps?: PaginationControlsProps
```
[the remark this answers is not in the document]
Comment (sksadjad) [reply packages/ssi-react/src/components/views/SSITableView/index.tsx]: renamed to pagination

File: packages/ssi-react/src/components/views/SSITableView/PaginationControls/index.tsx (hunk 1 above)
Comment (sksadjad) [reply packages/ssi-react/src/components/views/SSITableView/PaginationControls/index.tsx]: changed this to use translations. keys:
- `table_pagination_input_caption_go_to`
- `table_pagination_input_caption_page`

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**